patterns/components/contact_block.php

```diff
@@ -1,41 +1,84 @@
+      <!-- wp:image {"align":"center","id":471,"width":200,"height":200,"sizeSlug":"full","linkDestination":"none","className":"is-style-rounded"} -->
+      <figure
+        class="wp-block-image aligncenter size-full is-resized is-style-rounded"
+      >
+        <img
+          src="https://via.placeholder.com/150/002145/ffffff/"
+          alt=""
+          class="wp-image-471"
+          width="200"
+          height="200"
+        />
+      </figure>
+      <!-- /wp:image -->
+    </div>
+    <!-- /wp:column -->
 
-<!-- wp:column {"width":"40%"} -->
-<div class="wp-block-column" style="flex-basis:40%"><!-- wp:heading {"level":4,"style":{"typography":{"fontSize":"18px","letterSpacing":"-0.18px"}}} -->
-<h4 style="font-size:18px;letter-spacing:-0.18px">Contact</h4>
-<!-- /wp:heading -->
+    <!-- wp:column {"width":"40%"} -->
+    <div class="wp-block-column" style="flex-basis: 40%">
+      <!-- wp:heading {"level":4,"style":{"typography":{"fontSize":"18px","letterSpacing":"-0.18px"}}} -->
+      <h4 style="font-size: 18px; letter-spacing: -0.18px">Contact</h4>
+      <!-- /wp:heading -->
 
-<!-- wp:heading {"style":{"typography":{"fontSize":"30px","letterSpacing":"-0.3px"}}} -->
-<h2 style="font-size:30px;letter-spacing:-0.3px">FirstName LastName</h2>
-<!-- /wp:heading -->
+      <!-- wp:heading {"style":{"typography":{"fontSize":"30px","letterSpacing":"-0.3px"}}} -->
+      <h2 style="font-size: 30px; letter-spacing: -0.3px">
+        FirstName LastName
+      </h2>
+      <!-- /wp:heading -->
 
-<!-- wp:paragraph {"style":{"typography":{"fontSize":"16px"}}} -->
-<p style="font-size:16px">Title, Department</p>
-<!-- /wp:paragraph -->
+      <!-- wp:paragraph {"style":{"typography":{"fontSize":"16px"}}} -->
+      <p style="font-size: 16px">Title, Department</p>
+      <!-- /wp:paragraph -->
 
-<!-- wp:paragraph {"style":{"typography":{"fontSize":"16px"}}} -->
-<p style="font-size:16px">Email: <a href="mailto:name.name@ubc.ca">name.name@ubc.ca </a><br>Phone: 123.456.7890</p>
-<!-- /wp:paragraph --></div>
-<!-- /wp:column -->
+      <!-- wp:paragraph {"style":{"typography":{"fontSize":"16px"}}} -->
+      <p style="font-size: 16px">
+        Email: <a href="mailto:name.name@ubc.ca">name.name@ubc.ca </a
+        ><br />Phone: 123.456.7890
+      </p>
+      <!-- /wp:paragraph -->
+    </div>
+    <!-- /wp:column -->
 
-<!-- wp:column {"width":"35%"} -->
-<div class="wp-block-column" style="flex-basis:35%"><!-- wp:heading {"level":4,"style":{"typography":{"fontSize":"18px","fontStyle":"normal","fontWeight":"600","letterSpacing":"-0.18px"}}} -->
-<h4 style="font-size:18px;font-style:normal;font-weight:600;letter-spacing:-0.18px">Additional Information</h4>
-<!-- /wp:heading -->
+    <!-- wp:column {"width":"35%"} -->
+    <div class="wp-block-column" style="flex-basis: 35%">
+      <!-- wp:heading {"level":4,"style":{"typography":{"fontSize":"18px","fontStyle":"normal","fontWeight":"600","letterSpacing":"-0.18px"}}} -->
+      <h4
```

Comment:
Remove styles on item. Apply as site wide defaults. If a variation is needed create a variation and apply it.